etc/bin/run.sh

```diff
@@ -30,7 +30,10 @@ KERNEL_ASSEMBLY=`(cd ${PROG_HOME}/lib; ls -1 toree-kernel-assembly-*.jar;)`
 
 # disable randomized hash for string in Python 3.3+
 export PYTHONHASHSEED=0
+# The SPARK_OPTS values during installation are stored in TOREE_SPARK_OPTS. This allows values to be specified during
+# install, but also during runtime.
+export SPARK_OPTS="${TOREE_SPARK_OPTS} ${SPARK_OPTS}"
```

Comment:
Nah! Just wanted to make sure you gave that a try. So the only comment really for me is to somehow make the env var `TOREE_SPARK_OPTS` seem more like an internal thing. Not sure if adding `__` in front or something similar. I don't want to document this variable and it's only supported use is to exchange settings between `jupyter toree install` and `run.sh`. 

Thoughts?